inference-engine/src/multi_device/multi_device.cpp

```diff
@@ -258,8 +259,8 @@ void MultiDeviceExecutableNetwork::SetConfig(const std::map<std::string, Inferen
         assert(multiPlugin != nullptr);
         auto metaDevices = multiPlugin->ParseMetaDevices(priorities->second, {});
 
-        if (std::any_of(metaDevices.begin(), metaDevices.end(), [](const std::pair<DeviceName, DeviceInformation> & kvp) {
```

Comment:
kvp - key value pair. Use other name now

Reply:
this is minor, and would require changing/re-testing similar fix to the release branch (https://github.com/openvinotoolkit/openvino/pull/2251) which might miss the code-freeze then